src/components/radio/radio.ts

```diff
@@ -71,8 +71,8 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
   /** The currently selected radio button. Should match value. */
   private _selected: MdRadioButton = null;
 
-  /** Change event subscription set up by registerOnChange (ControlValueAccessor). */
-  private _changeSubscription: {unsubscribe: () => any} = null;
+  /** The method to be called in order to update ngModel */
+  private _onChangeHandler: (value: number) => void = (value) => {};
```

Comment:
1. I have no idea where `number` came from.
2. I'm for the rename - I was going to ask you to rename this for me since I had no idea what to call it.